### PR TITLE
Dev native

### PR DESCRIPTION
**Pull request by JoNil**
Use native raw sockets

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,9 +1,23 @@
-[workspace]
+[package]
+name = "pinger"
+version = "1.0.0"
+authors = ["Jonathan Nilsson <jonathan@voysys.se>"]
+edition = "2021"
+license = "MIT"
+description = "A small cross-platform library to execute the ping command and parse the output"
```
Comment (torkeldanielsson) [inline Cargo.toml]: is this still true?

File: src/bin/test.rs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+use pinger::ping_with_interval;
+use std::time::Duration;
+
+fn main() {
+    let pinger = ping_with_interval("google.com", Duration::from_secs(1), Some("enp5s0")).unwrap();
```
Comment (torkeldanielsson) [inline src/bin/test.rs:5]: hard coded network interface in test?

**Review by torkeldanielsson — APPROVED** [review]
some comments - this code is hard to parse so haven't reviewed all the details

File: src/bin/test.rs (hunk 2 above)
Comment (JoNil) [reply src/bin/test.rs:5]: Det är bra :D
